pkg/tsdb/cloudwatch/models/logs_query.go

```diff
@@ -1,28 +1,18 @@
 package models
 
-type LogGroup struct {
-	ARN       string `json:"arn"`
-	Name      string `json:"name"`
-	AccountID string `json:"accountId"`
-}
+import (
+	"github.com/grafana/grafana/pkg/tsdb/cloudwatch/kinds/dataquery"
+)
 
 type LogsQuery struct {
-	LogType            string `json:"type"`
-	SubType            string
-	Limit              *int64
-	Time               int64
-	StartTime          *int64
-	EndTime            *int64
-	LogGroupName       string
-	LogGroupNames      []string
-	LogGroups          []LogGroup `json:"logGroups"`
-	LogGroupNamePrefix string
-	LogStreamName      string
-	StartFromHead      bool
-	Region             string
-	QueryString        string
-	QueryId            string
-	StatsGroups        []string
-	Subtype            string
-	Expression         string
+	dataquery.CloudWatchLogsQuery
+	StartTime     *int64
+	EndTime       *int64
+	Limit         *int64
+	LogGroupName  string
+	LogStreamName string
+	QueryId       string
+	QueryString   string
+	StartFromHead bool
+	Subtype       string
```

Comment:
Note: previously there was a `SubType` and `Subtype` field to specify the log action to perform e.g. "StartQuery", "StopQuery", etc. Both of these are for the same field, but due to a previous refactoring, they were both added to the LogsQuery struct. I've removed the `SubType` field in favour of keeping it as `Subtype`.